dev/integration_tests/ios_platform_view_tests/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -54,6 +73,13 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		E09898E32853E9F000064317 /* Frameworks */ = {
```

Comment:
Was this empty frameworks build phase added accidentally? If Xcode did it automatically it's fine.

Reply:
it's automatically added by Xcode when I add the test target.